packages/teuchos/comm/src/Teuchos_StackedTimer.cpp

```diff
@@ -192,6 +192,8 @@ StackedTimer::computeColumnWidthsForAligment(std::string prefix,
   double total_time = 0.0;
 
   for (int i=0; i<flat_names_.size(); ++i ) {
+    if (sum_[i]/active_[i] < options.drop_time)
```

Comment:
Wouldn't you want `<= 0`, if the drop time is zero?

Reply:
I wanted to make sure that all timers print if drop tolerance is zero. It should never happen in practice but... Happy to change this if you want. I could then make the default -1.0 to force all timers to print.

Reply:
fixed.

Reply:
@rppawlo Thanks! :-D